core/controllers/editor.py

```diff
@@ -288,12 +288,15 @@ def _get_exploration_data(
                 exploration_id, exp_user_data.draft_change_list_exp_version)
             if exp_user_data and exp_user_data.draft_change_list_exp_version
             else None)
+        draft_id = (exp_user_data.draft_change_list_id if exp_user_data
+                    and exp_user_data.draft_change_list_id else 0)
```

Comment:
I'm a bit worried about all this conditioning on draft_change_list_id's existence. Can we maybe have a default=0 in core/storage/user/gae_models.py and then just treat it as though it always exists? That would also simplify some of the logic in #3560.